# Define the goal
float32 height
--- 
# Define the result
bool success
---
# Define a feedback message
float32 percent_complete